_posts/2018-05-17-Craft-Conf.md

```diff
@@ -0,0 +1,624 @@
+* Structure and clarity, everybody knew who was doing what and what they were working on
+* Dependability, they knew they could depend on each other
+* Psychological safety, the individuals in the team were able to make mistakes and take risks in front of each other
+
+It had nothing to do with IQ and individual talents of the people in the team and everything to do with how they worked together.
+
+#### Self-awareness
+95% of us think we are self-aware, while actually only 10-15% is!
+
+Self-awareness exercise:
+* What time do you naturally wake up?
+* When are your most productive hours?
+* When do you get your best ideas?
+* What does your ideal work day look like?
+
+We can use these answers to set up new ways of working within the team.
+
+We can do this during the project kickoff:
+1. Share team expertise, share what your role is and what you can bring to the table
+2. Clarify the roles, everyone needs to know who can do what work
+3. Talk about how you will work together, how you will meet, when you will meet, how to communicate, etc.
+
+### Organize better meetings and workshops
+Make your meetings count!
+* What is the purpose of the meeting?
+* What is the best format of the meeting? How long do we need? Do we need to split up the meeting?
+* Set meeting rhythms
+
+#### Amazon's two pizza teams:
+If a team is too large to be fed by two pizza's, then the team is just too large.
+
+#### Google Ventures Anxiety Parties
+Every three or four months, they would get together with a list of all the things that were bothering them about their performance and would share that with their team.
+An example of this might be that they weren't sure if they were spending too much time going to conferences or if they're working too slow.
+They would share these concerns during the anxiety party and the team members could respond whether they felt this actually was an issue or not to them.
+Someone could then find out that what they saw as an issue wasn't being considered an issue at all by the rest of the team, or perhaps even vice versa.
+These conversations build trust and better connections within the team.
+
+#### Take inspiration from workshops
+Great workshops have the following in common:
+* Collaboration
+* Creativity
+* Equal contribution
+* Good content
+* Clarity (on what needs to be done)
+* Motivation
+
+The next paperclip exercise shows you how to make your meetings more effective:
+First question Alison asked us: Come up with as many ideas as possible on what you can do with a paperclip
+Second question: Which idea would make a good business proposition?
+
+Two different activities came up during this exercise:
+* **Divergent thinking**: Trying to come up with as many ideas as possible, the focus was on the quantity.
+* **Convergent thinking**: Applying criteria to your ideas and being critical.
+
+It's very important to keep these two activities separate during your meetings.
+We also need productive conflict and avoid group thinking!
```

Comment:
maybe rework this, phrasing is a little weird
Is it meant to say that it needs productive conflict thinking and need to avoid group thinking?